src/Compilers/CSharp/Portable/Binder/Semantics/Operators/UnaryOperatorOverloadResolution.cs

```diff
@@ -319,6 +320,36 @@ private bool GetUserDefinedOperators(UnaryOperatorKind kind, BoundExpression ope
                 }
             }
 
+            // Look in base interfaces, or effective interfaces for type parameters  
```

Comment:
A note above for the spec quote could be useful to indicate that it's no longer valid.